image/src/mcclimatevickithermostat.json

```diff
@@ -0,0 +1,13 @@
+{
+  "commands": [
+    {
+      "command": {
+        "originalName": "setTemperatur",
```

Comment:
Not missing 'e' at the end? setTemperatur -> setTemperature?

Reply:
it does, but thinkmoto used setTemperatur in mongoDB that's why those 2 have to match, otherwise I'll have to ask them to update DB again. I can do this later, when we start doing changes on the application.